sdk/schemaregistry/schema-registry-avro/test/public/avroSerializer.spec.ts

```diff
@@ -32,6 +39,10 @@ describe("AvroSerializer", async function () {
     };
   });
 
+  afterEach(async function (this: Context) {
+    await removeSchema();
```

Comment:
Could we instead update every unit test to add the names of the schemas they register into an array and then delete those schemas here?

Reply:
My concern is when multiple tests register the same schema, only one version is actually on the portal. So there'll be more delete calls than the number of schemas to delete.

Reply:
The test could just register a single name in this case, no?